contracts/GaugeController.sol

```diff
@@ -63,10 +141,21 @@ contract GaugeController is IGaugeController {
         gaugeTwab.details = twabDetails;
     }
 
+    /// @TODO: Add Governance/Executive authorization modifier/function.
+    /**
+     * @notice Add new gauge with "1e18" scale to the GaugeController.
+     * @param _gauge Address of the Gauge
+     */
     function addGauge(address _gauge) public {
         addGaugeWithScale(_gauge, 1 ether);
```

Comment:
We should add a `_addGaugeWithScale` internal function to avoid calling a public one.